src/Symfony/Component/Form/Tests/Extension/Core/Type/RepeatedTypeTest.php

```diff
@@ -78,6 +79,42 @@ public function testSetRequired()
         $this->assertFalse($form['second']->isRequired());
     }
 
+    public function testMappedOverridesDefault()
+    {
+        $form = $this->factory->create(NotMappedType::class);
+        $this->assertFalse($form->getConfig()->getMapped());
+
+        $form = $this->factory->create(static::TESTED_TYPE, null, [
+            'type' => NotMappedType::class,
+        ]);
+
+        $this->assertTrue($form['first']->getConfig()->getMapped());
+        $this->assertTrue($form['second']->getConfig()->getMapped());
+    }
+
+    /**
+     * @param string $configurationKey
```

Comment:
please remove this line

Reply:
Done!

Just a side question in case you have time: Wouldn't such comments ease moving to typehinted scalar types when 3.4 will be EOL and 4.4 will require 7.1?

Reply:
Tests are not covered by BC promise, we do add those in the code base though.

Reply:
We already added all possible types in 5.0, we should not forget to add it here also that true; but if we do, that's no big deal (and eventually someone will figure out :) )